clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -306,6 +306,7 @@ private boolean renameDirectory(Path src, Path dst) throws IOException {
 
         ListingIterator iterator = new ListingIterator(src, true, listAmount);
         while (iterator.hasNext()) {
+            //TODO (Tals): parallelize objects rename process.
```

Comment:
nit:

```suggestion
            // TODO (Tals): parallelize objects rename process.
```

(I follow https://google.github.io/styleguide/cppguide.html#TODO_Comments mostly)